lib/InvoiceItem.php

```diff
@@ -23,6 +23,7 @@
  * @property string|null $subscription
  * @property string $subscription_item
  * @property array|null $tax_rates
+ * @property bool $unified_proration
```

Comment:
That one shouldn't be here, ugh. If you can, I'd rather you remove it (and I flagged internally that it should be hidden)